services/navigation/builtin/builtin.go

```diff
@@ -438,3 +475,27 @@ func (svc *builtIn) GetObstacles(ctx context.Context, extra map[string]interface
 	defer svc.mu.RUnlock()
 	return svc.obstacles, nil
 }
+
+func newFilePathLoggerConfig(filepath string) zap.Config {
```

Comment:
Surely this should live somewhere more generic than in the nav service

Reply:
The thing is that this is a stopgap measure that is intended to be used by the motion team. I am kinda of hesitant to put it in a more generic place b/c doing so implies that it is in some way canonical (which it is not). 

The long-term solution to this is for users to tell the viam-server to only output logs from specific components/services but we don't have that feature currently.

Reply:
done